pyuoi/linear_model/logistic.py

```diff
@@ -616,9 +617,19 @@ def _logistic_regression_path(X, y, Cs=48, fit_intercept=True,
         target = Y_multi
         if penalty == 'l2':
             w0 = w0.ravel()
+            if coef_mask is not None:
+                x0 = np.zeros_like(w0)
 
             def func(x, *args):
-                return _multinomial_loss_grad(x, *args)[0:2]
+                X, Y, _, mask, _ = args
```

Comment:
What's the purpose of assigning X and Y here?

Reply:
I used them in an old version. I'll fix this and below.